src/index.ts

```diff
@@ -139,6 +141,27 @@ class MetricsGatherer {
 		};
 	}
 
+	aggregateRequestHandler(authTest? : AuthTestFunc, callback? : Function) : express.Handler {
+		const aggregatorRegistry = new prometheus.AggregatorRegistry();
+		return (req, res) => {
+			if (authTest && !authTest(req)) {
+				res.status(403).send();
```

Comment:
This should have a return because as it stands this will still go through to call `aggregatorRegistry.clusterMetrics()`

Reply:
Good catch